src/vanilla/ClientModelExtensions.cs

```diff
@@ -346,7 +347,7 @@ public static string CheckNull(string valueReference, string executionBlock)
             if (sequence != null && sequence.ShouldValidateChain())
             {
                 var elementVar = scope.GetUniqueName("element");
-                var innerValidation = sequence.ElementType.ValidateType(scope, elementVar, null);
+                var innerValidation = sequence.ElementType.ValidateType(scope, elementVar, isNullable, null);

```

Comment:
I don't think that nullable collection means nullable element, does `sequence.ElementType` have IsNullable()?

Reply:
No, `sequence.ElementType` doesn't have `IsNullable()` but only `sequence` has `sequence.IsNullable`...

Reply:
let's just pass `false` then. To keep the existing behavior.